src/Commands/Parameters/StringLiteralParameter.ts

```diff
@@ -0,0 +1,23 @@
+import { Parameter } from "./Parameter";
+
+/**
+ * A string literal parameter.
+ */
+export class StringLiteralParameter extends Parameter {
+    /**
+     * The name of the string literal.
+     */
+    public name: string;
+
+    /**
+     * Initializes a new instance of the Parameter class.
```

Comment:
I still haven't found the parameter typo -- would you mind pointing me to which one is misspelled?